Gemfile

```diff
@@ -3,3 +3,7 @@ source "https://rubygems.org"
 gemspec
 
 gem "builder", ">= 2.1.2"
+
+# HTTP.rb (used by solr_wrapper to download solr for integration testing) fails
+# to download the full contents of files (under jruby)?
+gem "http", '< 5', platforms: :jruby
```

Comment:
This is confusing to me -- do you mean http 5 is what fails to download, and http 4 was ok, is why we're limiting to http 4 here? Should the comment say that?

This probably isn't tenable as a long-term solution, but okeydoke.